curriculum/js1/7.js

```diff
@@ -5,8 +5,10 @@
  * @returns {boolean}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, b=a-1) => {
+  if (b===1) return true
+  if ((a%b)===0) return false;
+  return solution(a, b-1)
```

Comment:
Please add a semicolon to the end of this return statement.

Reply:
Thank you! I just made the changes. Could you please review again? Appreciate it!